service-workers/service-worker/claim-shared-worker-fetch.https.html

```diff
@@ -30,9 +30,13 @@
                                          'fetch() should not be intercepted.'))
     // Register a service worker.
     .then(() => service_worker_unregister_and_register(t, script, scope))
-    .then(r => worker = r.installing)
-    .then(() => wait_for_state(t, worker, 'activated'))
+    .then((r) => {
```

Comment:
nit: here too, no () for consistency with rest of file

Reply:
Done